packages/terra-functional-testing/src/commands/hide-input-caret.js

```diff
@@ -0,0 +1,20 @@
+const Logger = require('../logger/logger');
+
+/**
+ * Hides the blinking input caret that appears within editable text areas to prevent inconsistent test failures.
+ *
+ * @param {string} selector The selector for the element to hide the caret of
+ */
+const hideInputCaret = (selector) => {
+  try {
+    global.browser.execute(`document.querySelector("${selector.replace(/"/g, '\\"')}").style.caretColor = "transparent";`);
+  } catch (error) {
+    if (!global.browser.isExisting(selector)) {
+      throw Logger.error(`No element could be found with the selector '${selector}'.`);
```

Comment:
I'd be interested in what the catch error message is. Is this if/else block necessary if we're always throwing an error?

```js
} catch (error) {
  throw error;
}
```

Reply:
I think this was a mistake. I updated it [here](https://github.com/cerner/terra-toolkit/pull/438/commits/4d600076e3df0e9c99b327e4e0be28289810fa0a) to just log the message instead of throwing the message. If the selector does not exist, which is a common case, we would get a message like this `javascript error: Cannot read property 'style' of null`. I think logging that specific message is more helpful. Other types of error would just get thrown as is.